src/main/kotlin/calculator/ChunkStringCalculator.kt

```diff
@@ -0,0 +1,42 @@
+package calculator
+
+import java.util.LinkedList
+
+class ChunkStringCalculator(wholeElements: List<String>) {
+    private val calculationDeque = LinkedList<String>()
+
+    init {
+        calculationDeque.addAll(wholeElements)
+    }
+
+    fun calculate(): String {
+        val operand1: String = calculationDeque.pollFirst()
+        val operator: String = calculationDeque.pollFirst()
+        val operand2: String = calculationDeque.pollFirst()
+
+        try {
+            operand1.toInt()
+            operand2.toInt()
+        } catch (e: NumberFormatException) {
+            throw NumberFormatException("숫자형이 아닌 피연산자가 들어왔다.")
+        }
+
+        val chunkResult = when (Operator.fromSymbol(operator)) {
+            Operator.PLUS -> Operator.PLUS.calcFunc(operand1.toInt(), operand2.toInt())
+            Operator.MINUS -> Operator.MINUS.calcFunc(operand1.toInt(), operand2.toInt())
+            Operator.MULTIPLY -> Operator.MULTIPLY.calcFunc(operand1.toInt(), operand2.toInt())
+            Operator.DIVIDE -> Operator.DIVIDE.calcFunc(operand1.toInt(), operand2.toInt())
+        }.toString()
```

Comment:
Operator Enum 을 잘 정리해주셨기에, 다음과 같이 작성해볼 수도 있겠네요!

```suggestion
        val chunkResult = Operator.fromSymbol(operator).calcFunc(operand1.toInt(), operand2.toInt()).toString()
```

Reply:
다음 [커밋](https://github.com/next-step/kotlin-racingcar/pull/1356/commits/c5956546a916d10852f0e8a964e1b75bc8feba9f)에서 반영하였습니다. 감사합니닷!